drivers/video/gc2145.c

```diff
@@ -42,6 +42,54 @@ LOG_MODULE_REGISTER(video_gc2145, CONFIG_VIDEO_LOG_LEVEL);
 #define GC2145_REG_SUBSAMPLE_MODE       0x9A
 #define GC2145_SUBSAMPLE_MODE_SMOOTH    0x0E
 
+/* MIPI-CSI registers - on page 3 */
+#define GC2145_REG_DPHY_MODE1		0x01
+#define GC2145_DPHY_MODE1_CLK_EN	BIT(0)
+#define GC2145_DPHY_MODE1_LANE0_EN	BIT(1)
+#define GC2145_DPHY_MODE1_LANE1_EN	BIT(2)
+#define GC2145_DPHY_MODE1_CLK_LANE_P2S_SEL	BIT(7)
+
+#define GC2145_REG_DPHY_MODE2	0x02
+#define GC2145_DPHY_MODE2_CLK_DIFF(a)		((a) & 0x07)
+#define GC2145_DPHY_MODE2_LANE0_DIFF(a)	(((a) & 0x07) << 4)
+
+#define GC2145_REG_DPHY_MODE3	0x03
+#define GC2145_DPHY_MODE3_LANE1_DIFF(a)	((a) & 0x07)
+#define GC2145_DPHY_MODE3_CLK_DELAY		BIT(4)
+#define GC2145_DPHY_MODE3_LANE0_DELAY		BIT(5)
+#define GC2145_DPHY_MODE3_LANE1_DELAY		BIT(6)
+
+#define GC2145_REG_FIFO_FULL_LVL_LOW	0x04
+#define GC2145_REG_FIFO_FULL_LVL_HIGH	0x05
+#define GC2145_REG_FIFO_MODE		0x06
+#define GC2145_FIFO_MODE_READ_GATE	BIT(3)
+#define GC2145_FIFO_MODE_MIPI_CLK_MODULE	BIT(7)
+
+#define GC2145_REG_BUF_CSI2_MODE	0x10
+#define GC2145_CSI2_MODE_DOUBLE		BIT(0)
+#define GC2145_CSI2_MODE_RAW8		BIT(2)
+#define GC2145_CSI2_MODE_MIPI_EN	BIT(4)
+#define GC2145_CSI2_MODE_EN		BIT(7)
+
+#define GC2145_REG_MIPI_DT		0x11
+#define GC2145_REG_LWC_LOW		0x12
+#define GC2145_REG_LWC_HIGH		0x13
+#define GC2145_REG_DPHY_MODE		0x15
+#define GC2145_DPHY_MODE_TRIGGER_PROG	BIT(4)
+
+#define GC2145_REG_FIFO_GATE_MODE	0x17
+#define GC2145_REG_T_LPX		0x21
+#define GC2145_REG_T_CLK_HS_PREPARE	0x22
+#define GC2145_REG_T_CLK_ZERO		0x23
+#define GC2145_REG_T_CLK_PRE		0x24
+#define GC2145_REG_T_CLK_POST		0x25
+#define GC2145_REG_T_CLK_TRAIL		0x26
+#define GC2145_REG_T_HS_EXIT		0x27
+#define GC2145_REG_T_WAKEUP		0x28
+#define GC2145_REG_T_HS_PREPARE		0x29
+#define GC2145_REG_T_HS_ZERO		0x2a
+#define GC2145_REG_T_HS_TRAIL		0x2b
+
```

Comment:
I will try to port it to the CCI library, I [got started for DVP](https://github.com/zephyrproject-rtos/zephyr/commit/7b9d6af7d26bcc4930fa2521be232b399d42c512) and will do it after the merge